src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Bmi2.PlatformNotSupported.cs

```diff
@@ -61,8 +70,16 @@ public abstract class X64
         /// <summary>
         /// unsigned int _mulx_u32 (unsigned int a, unsigned int b, unsigned int* hi)
         ///   MULX r32a, r32b, reg/m32
+        /// The above native signature does not directly correspond to the managed signature. 
```

Comment:
Is it me or the comment above is wrong? It shows a `unsigned int* hi` parameter.

Reply:
This PR changes the APIs to return “high”, which is different from C++.